x-pack/legacy/plugins/alerting/server/alerts_client.ts

```diff
@@ -171,12 +174,17 @@ export class AlertsClient {
       });
       createdAlert.attributes.scheduledTaskId = scheduledTask.id;
     }
-    return this.getAlertFromRaw(createdAlert.id, createdAlert.attributes, references);
+    return this.getAlertFromRaw(
+      createdAlert.id,
+      createdAlert.attributes,
+      createdAlert.updated_at,
```

Comment:
One thing worth noting in regards to comment https://github.com/elastic/kibana/issues/52738#issuecomment-569234567: This won't allow users to sort on the last time the alert was updated.

I'm not sure if this is a requirement but if ever it becomes, a separate PR to add `updated_at` to the following file is all we'd have to do: https://github.com/elastic/kibana/blob/e233e419cf468a1c3dc5724733e735da0bd449ce/src/core/server/saved_objects/service/lib/search_dsl/sorting_params.ts#L23

Reply:
Ah, that's an interesting point.
Thank's for pointing it out.